pulpcore/cli/common.py

```diff
@@ -42,13 +55,11 @@ def wait_for_task(self, task_href, timeout=120):
                 raise click.ClickException("Task failed")
             if task["state"] == "canceled":
                 raise click.ClickException("Task canceled")
-            time.sleep(1)
-            timeout -= 1
             click.echo(".", nl=False, err=True)
         raise click.ClickException("Task timed out")
 
 
-def _config_callback(ctx, param, value):
+def _config_callback(ctx: click.Context, param: Any, value: str) -> None:
```

Comment:
I think param could maybe be a str but I guess it doesn't make a huge difference since we're not using it.

Reply:
I think, `str` didn't work...